aeron-driver/src/main/java/io/aeron/driver/MediaDriver.java

```diff
@@ -464,6 +464,7 @@ public static final class Context extends CommonContext
         private long publicationConnectionTimeoutNs = Configuration.publicationConnectionTimeoutNs();
         private long publicationLingerTimeoutNs = Configuration.publicationLingerTimeoutNs();
         private long untetheredWindowLimitTimeoutNs = Configuration.untetheredWindowLimitTimeoutNs();
+        private long untetheredLingerTimeoutNs = Configuration.untetheredLingerTimeoutNs();
```

Comment:
I think would be simpler implementation if we set this value to be Aeron.NULL_VALUE initially, then check for that value during the `MediaDriver.Context::conclude` call and set it to the window timeout there.